catalogue/openapi-config/openapi-config.yml

```diff
@@ -8,3 +9,7 @@ additionalProperties:
   serializationLibrary: kotlinx_serialization
   supportAndroidApiLevel25AndBelow: true
   useCoroutines: true
+files:
+  Catalogue.kt.mustache:
+    templateType: SupportingFiles
+    destinationFilename: src/main/kotlin/com/tidal/sdk/catalogue/generated/Catalogue.kt
```

Comment:
generated code should not be here imho. check out where dagger places generated files.

Reply:
I'll read up on the conventions